tools/maplint/lints/atmos_var_edits.yml

```
@@ -0,0 +1,4 @@
+help: "This var is obsolete/no longer in use. Please remove it."
```

I would not say this var is "obsolete", far from it, it's necessary for all of these to function. 
Something more accurate would be
```suggestion
help: "Please replace this var edit with an existing layer subtype, or create a layer subtype if one does not exist."
```